vita3k/main.cpp

```diff
@@ -186,6 +186,7 @@ int main(int argc, char *argv[]) {
     }
 
     Ptr<const void> entry_point;
+    host.io.title_id = host.app_title_id;
```

Comment:
delete this, broken boot for some case.
look code before, also, io.title_id is already set

Reply:
done

Reply:
thx, sorry for this, i try fix myself this issue